### PR TITLE
Feature/df 1060  work effort tests

### PR DESCRIPTION
**Pull request by emina-str**

### CODE REVIEW

File: application/pom.xml [hunk 1]
```diff
@@ -27,6 +27,11 @@
             <groupId>org.springframework.security</groupId>
             <artifactId>spring-security-core</artifactId>
         </dependency>
+        <dependency>
```
Comment (pavax) [inline application/pom.xml]: removed

File: application/pom.xml [hunk 2]
```diff
@@ -27,6 +27,11 @@
...
+            <groupId>org.springframework.security</groupId>
```
Comment (pavax) [inline application/pom.xml]: removed

File: application/pom.xml [hunk 3]
```diff
@@ -44,6 +49,11 @@
             <version>${project.version}</version>
             <classifier>tests</classifier>
         </dependency>
+        <dependency>
```
Comment (pavax) [inline application/pom.xml]: removed as not needed

File: application/src/main/java/ch/admin/seco/alv/service/onlineform/application/npa/ProofOfWorkEffortsApplicationService.java [hunk 4]
```diff
@@ -133,7 +124,7 @@ public ProofOfWorkEffortsDto create(CreateProofOfWorkEffortsDto createProofOfWor
 
 	@PreAuthorize("hasRole('ROLE_SYSADMIN') or @authorizationService.isCurrentUser(#userId)")
 	public Page<ProofOfWorkEffortsDto> findByOwnerUserId(UserId userId, int pageNumber) {
-		PageRequest pageRequest = PageRequest.of(pageNumber, SIZE);
+		PageRequest pageRequest = PageRequest.of(pageNumber, SIZE, Sort.by(desc("controlPeriod.value"), desc("updatedAt"), desc("createdAt")));
```
Comment (pavax) [inline application/src/main/java/ch/admin/seco/alv/service/onlineform/application/npa/ProofOfWorkEffortsApplicationService.java]: @emina-str why do we sort also against the updatedAt and createdAt? is this realy needed?

Comment (emina-str) [reply application/src/main/java/ch/admin/seco/alv/service/onlineform/application/npa/ProofOfWorkEffortsApplicationService.java]: @pavax  If we can be sure there will be only unique values for control-period, it is not necessary. I'll remove it.

**Pull request merged**